app/ai/voice/agents/breeze_buddy/template/ui_catalog.py

```
@@ -261,19 +261,29 @@ class QuickReplyItem(BaseModel):
+
+
 class QuickReplies(_CatalogBase):
     """Turn-scoped quick-reply row — the LLM-driven equivalent of startup
     chicklets.
 
-    Emit at the end of any assistant turn where you want to suggest 2–5
-    follow-up options. The widget renders them as pill buttons right below
-    the message; clicking one fires a ``to_assistant`` action and disables
-    the whole row (one-shot).
+    Emit at the end of an assistant turn to offer tappable follow-ups or a
+    short option/category menu (2–12 pills). The widget renders them as pill
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)